flowcore/src/main/java/com/raizlabs/android/dbflow/annotation/Action.java

```diff
@@ -0,0 +1,34 @@
+package com.raizlabs.android.dbflow.annotation;
+
+/**
+ * Actions associated with on update and on delete
+ */
+public enum Action {
```

Comment:
Not sure if the name is descriptive enough. I would either do ```FKAction``` or ```ForeignKeyAction``` (I like this one).